lib/stoplight.rb

```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
 
+require "zeitwerk"
+
+loader = Zeitwerk::Loader.for_gem
+loader.inflector.inflect("io" => "IO")
+loader.ignore("#{__dir__}/stoplight/rspec.rb", "#{__dir__}/stoplight/rspec")
```

Comment:
Should we also ignore `bench/**`?

Reply:
It’s out of the `lib` directory so ignored by default. If it was not ignored, it would raise a Zeitwerk error -- Bench constant not found or something like this. 

Just to be cleal, here ignore shared RSpec helpers for user-defined notifiers, not specs for Stoplight itself.